packages/api-docs-builder/buildApiUtils.ts

```diff
@@ -9,6 +9,28 @@ import { replaceComponentLinks } from './utils/replaceUrl';
 import findPagesMarkdown from './utils/findPagesMarkdown';
 import { TypeScriptProject } from './utils/createTypeScriptProject';
 
+/**
+ * TODO: this should really be fixed in findPagesMarkdown().
+ * Plus replaceComponentLinks() shouldn't exist in the first place,
+ * the markdown folder location should match the URLs.
```

Comment:
The logic that converts docs markdown file location to URL location is a mess right now. One step toward making it straightforward.